pkg/graph/resolvers/governance_request_feedback.go

```diff
@@ -13,3 +13,17 @@ import (
 func GetGovernanceRequestFeedbacksByIntakeID(ctx context.Context, store *storage.Store, id uuid.UUID) ([]*models.GovernanceRequestFeedback, error) {
 	return store.GetGovernanceRequestFeedbacksByIntakeID(ctx, id)
 }
+
+// GetGovernanceRequestFeedbackAuthor returns the full user info for a feedback item's author
+func GetGovernanceRequestFeedbackAuthor(
+	ctx context.Context,
+	fetchUserInfo func(context.Context, string) (*models.UserInfo, error),
+	feedbackAuthorEUAID string,
+) (*models.UserInfo, error) {
+	authorInfo, err := fetchUserInfo(ctx, feedbackAuthorEUAID)
+	if err != nil {
+		return nil, err
+	}
+
+	return authorInfo, nil
+}
```

Comment:
Might be nice to use the UserInfo dataloader here, since this could easily be requested multiple times in a row when fetching multiple feedbacks in an array (I think the dataloader will help with that?)

https://github.com/CMSgov/easi-app/blob/8c0fe2bcfbbd80cdbad0f6cbe095f8e45c6d65dd/pkg/dataloaders/user_info_loader.go

https://github.com/CMSgov/easi-app/blob/472f21cc8c8ec5e0f0446ede361ee61363c3e291/pkg/graph/resolvers/trb_request.go#L407-L423

Reply:
Nice, I didn't realize we had that. I'll take a look at it.

Reply:
Implemented in https://github.com/CMSgov/easi-app/pull/2125/commits/55d0e724b0364ce15a28940a1319413111df7d3d.